### PR TITLE
refa: some minor refactoring

### PR DESCRIPTION
**Pull request by dynox**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,217 +1,209 @@
...
                 else:
-                    isins.append('NA')
+                    isins.append("NA")
 
             except Exception as e:
                 print(e)
 
-    dataframe.loc[:, 'isin'] = isins
+    dataframe.loc[:, "isin"] = isins
     print(dataframe)
```
Comment (dennislemonmarkets) [inline main.py:112]: print head only if necessary or remove it

File: main.py [hunk 2]
```diff
@@ -1,217 +1,209 @@
...
+def trade_decision(dataframe: pd.DataFrame):
     buy = []
     sell = []
     for index, row in dataframe.iterrows():
         # if sentiment higher than 0.5 and ISIN present, place ISIN in buy list
-        if row['score'] > 0.5 and row['isin'] != 'NA':
-            print(f'Buy {row["ticker"]} ({row["isin"]}) with sentiment score {row["score"]}.')
-            buy.append(row['isin'])
+        if row["score"] > 0.5 and row["isin"] != "NA":
+            print(
```
Comment (dennislemonmarkets) [inline main.py:122]: how about using logging.info and setting it up with `logging.basicConifg()`

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 21.9b0
+    hooks:
```
Comment (dennislemonmarkets) [inline .pre-commit-config.yaml:4]: Last comment: I would remove this again. It does not provide any value for the content

Comment (llimona) [reply .pre-commit-config.yaml:4]: @dynox please remove this

**Pull request merged**